_posts/2023-02-16-secure-cloud-foundation.md

```diff
@@ -0,0 +1,209 @@
+
+## The Container Detective: Trivy
+
+Trivy is an open-source cli tool provided by Aqua Security.
+
+### Container scanning
+
+Trivy can scan container images against well known vulnerabilities.
+On the [tool's homepage](https://trivy.dev/), you can enter public available docker hub images to test it out.
+
+### Dependency scanning
+
+Trivy can scan your dependencies for well known vulnerabilities.
+It has a mode that automatically discovers, declarations files for various package managers
+
+### CI/CD integration
+
+Because Trivy is a cli tool it can easily be integrates in CI/CD pipelines.
+Trivy also maintains a [GitHub action](https://github.com/aquasecurity/trivy-action) to integrate it in GitHub actions.
+But the community has created 2 additional GitHub actions.
+
+### AWS integration
+
+Trivy can be run locally to scan your AWS environment using the AWS cli.
+The default included check scans against AWS CIS 1.2.0 benchmark.
+It shows summarizes a lists of issues, and gives description of how to resolve the issue, it won't automatically fix it.
+
+### Secret scanning
+
+Trivy can scan your code for secrets, it can scan for AWS access key, GCP service account, GitHub personal access token,
+GitLab personal access token, Slack access token, etc.
+
+It can do this either on the file system or inside a container image.
+
+### Configuration issues
+
+Trivy can scan your configuration files like Dockerfiles, Kubernetes manifests, Terraform, CloudFormation, etc.
+against known configuration issues
+
+### Custom policies
+
+For all the mentioned functionality custom policies can be written in Rego.
+
+## The Composer: Fugue
+
+Fugue is a cloud security platform that helps you to secure your cloud environment,
+it was bought by Snyk some time ago and after this take-over Snyk started working on Snyk Cloud.
+
+### Baseline enforcement
+
+Fugue allows you to take a snapshot of your Cloud environment and use it as baseline.
+
+This prevents anyone from making modifications to your environment that are not compliant with your baseline.
+
+It can't recreate or delete resources it only enforces by modifying them back to the original state of the snapshot.
+
+### Policy scanning
+
+Fugue allows you to write policies to scan your AWS environment for compliance,
+or you can use one of the pre-defined policies like CIS AWS Foundations Benchmark.
+
+It does not provide automatic solutions to fix the violations, but has descriptions on how to fix them.
+
+### CI/CD integration
+
+Fugue can be integrated with your CI/CD pipeline to scan your infrastructure as code for compliance using their cli.
+
+They have a guide on how to set this up with CircleCI,
+but it should be possible to set this up with any other CI/CD tools.
+
+### Custom policies
+
+Fugue allows you to write custom policies in Rego.
+
+<!---
+TO DO -- 
+
+Add images for each section.
+
+Enrich post with links
+
+Check for spelling/grammar erros
+
+Get banner image
+-->
```

Comment:
Add a conclusion. 

It's clear that Rego is the compliance configuration language of choice for this type of tooling (it's used / supported by almost all of the ones you looked at). Explain what the difference is between the tools (use the references to the nicknames here as they seem to suggest what they can and cannot do). 

Include in the conclusion how these solutions compare to the native solutions of the cloud providers (AWS security hub, Azure security center / Sentinel)

Add a call to action to either look into one of the tools, follow the blog for more details looks at the tools in question or reach out to one of you (LinkedIn? or refer to Ordina, as you like). 

Add any interesting links, at least links to all of the websites of the tools / products mentioned.